Provide getting started document for VS Code

jdneo opened the pull request:
Signed-off-by: sheche <sheche@microsoft.com>

jdneo commented:
Hi @jperedadnr,

This is the PR to initialize the getting started document for VS Code.

One question: I noticed that the existing screenshots all have a white colored margin as the background, the screenshots in this PR does not have that, do I need to add them?

jdneo commented:
@jperedadnr would you mind to take a look at this PR?

jperedadnr commented:
Ok, we'll look into it.
About screenshots, we typically take them on macOS, which adds this shadow effect, but it is not mandatory.

abhinayagarwal commented:
The screenshots are very close to the paragraph text. Can you add the following style to `gluon.css`:

```
/* Visual Code */
#IDE-VSCode img {
    padding: 1em;
}
```

abhinayagarwal commented on ide-vscode.html:
```diff
@@ -0,0 +1,575 @@
+
+    <a href="images/ide/vscode/gradle/vscode03.png" target="_blank"><img src="images/ide/vscode/gradle/vscode03.png" alt="Open the project"/></a>
+
+    you can remove the <kbd>app/src/test</kbd> folder if you don't want to add tests for the project.
```
s/you can remove the/Remove the

abhinayagarwal commented on ide-vscode.html:
```diff
@@ -0,0 +1,575 @@
+
+    <a href="images/ide/vscode/gradle/vscode04.png" target="_blank"><img src="images/ide/vscode/gradle/vscode04.png" alt="Update the project"/></a>
+
+    And you will get the JavaFX dependencies after the update.
```
A better way to put this would be "Visual Studio will list all the JavaFX dependencies after the update"

abhinayagarwal commented on ide-vscode.html:
```diff
@@ -0,0 +1,575 @@
+
+    <a href="images/ide/vscode/gradle/vscode05.png" target="_blank"><img src="images/ide/vscode/gradle/vscode05.png" alt="Gradle dependencies"/></a>
+
+    As for any other Gradle dependencies, these jars can be found in the local <kbd>.gradle</kbd> repository.
```
I don't get this statement. What other Gradle dependencies are we referring to?

jdneo replied:
I copied this from the Eclipse tutorial, see:
- `Non-modular with Maven` > `2. Verify the pom`
- `Non-modular with Gradle` > `2. Modify the build`

I guess it intends to tell the readers that the build tool will manage all the dependencies from different projects into those folders?

I agree that this statement somehow introduces confusions and I'm ok to remove it.

abhinayagarwal replied:
In that case I would keep these as it is and update the sentences for all IDE together in a later PR.

abhinayagarwal commented on ide-vscode.html:
```diff
@@ -0,0 +1,575 @@
+    
+    <a href="images/ide/vscode/maven/vscode04.png" target="_blank"><img src="images/ide/vscode/maven/vscode04.png" alt="Update the pom"/></a>
+   
+    As for any other maven dependencies, these jars can be found in the local <kbd>.m2</kbd> repository.
```
I don't get this statement. What other Maven dependencies are we referring to?

abhinayagarwal commented on ide-vscode.html:
```diff
@@ -0,0 +1,575 @@
+
+    <a href="images/ide/vscode/modular/gradle/vscode05.png" target="_blank"><img src="images/ide/vscode/modular/gradle/vscode05.png" alt="Gradle dependencies"/></a>
+
+    As for any other Gradle dependencies, these jars can be found in the local <kbd>.gradle</kbd> repository.
```
I don't get this statement. What other Gradle dependencies are we referring to?

abhinayagarwal reviewed (commented):
Overall the PR LGTM. Let a few minor comments.

jdneo commented:
@abhinayagarwal Thank you for reviewing the PR. I've addressed some of the comments.

As of the statements of `other Maven dependencies` and `other Gradle dependencies`, I'm ok to remove them.

Waiting for your suggestion and I'll make follow-up actions if needs.
